pkgs/top-level/all-packages.nix

```diff
@@ -34556,4 +34556,6 @@ with pkgs;
   honeytail = callPackage ../servers/tracing/honeycomb/honeytail { };
 
   honeyvent = callPackage ../servers/tracing/honeycomb/honeyvent { };
+
+  plasma-theme-switcher = callPackage ../applications/misc/plasma-theme-switcher {};
```

Comment:
we don't just add to the end of `all-packages.nix`

also, this could/should use `libsForQt5.callPackage`

Reply:
In what order is the package supposed to be added? I've tried finding alphabetical order first, but with no success.

Please excuse my question, this is my first time working with/on nix

Reply:
There's a comment at the top of the file trying to explain how it's supposed to be sorted, but it's quite a mess, tbh

Reply:
I've tried to sort it in near a similar application now, I honestly missed the comment first time, sorry for that.

Let me know if the new sorting works! :)